blog.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Blog
 layout: blog
+permalink: blog
```

Comment:
I know this is my fault, not yours, but as long as you're editing this front matter, would be willing to make the pages have the fields in the same order?